sopel/loader.py

```diff
@@ -44,7 +45,7 @@ def clean_callable(func, config):
     prefix = config.core.prefix
     help_prefix = config.core.help_prefix
     func._docs = {}
-    doc = trim_docstring(func.__doc__)
+    doc = trim_docstring(inspect.getdoc(func))
```

Comment:
@Exirel, you think this is fine for 7.1? I'd also deprecate `trim_docstring()` for 7.1, and remove it in 8.0.

```suggestion
    doc = inspect.getdoc(func)
```

Reply:
With a bit more code, yes:

* `inspect.getdoc(func)` will return `None` if `func` doesn't have a docstring
* `inspect.getdoc(func)` will return a string, while `trim_docstring` will return a list

However, since `getdoc` (using `cleandoc`) will remove empty line before/after the docstring, you shouldn't need to use `trim_docstring` at all, and do this:

```suggestion
    doc = []
    docstring = inspect.getdoc(func)
    if docstring:
        doc = docstring.splitlines()
```

Reply:
Adopted this approach in a70b1e6f653d4cac9ef8964d60b01943d284068f.